src/pages/storage/StorageVolumes.tsx

```diff
@@ -65,42 +77,65 @@ const StorageVolumes: FC = () => {
   }
 
   const headers = [
-    { content: "Name", sortKey: "name" },
-    { content: "Pool", sortKey: "pool" },
-    { content: "Type", sortKey: "type" },
-    { content: "Content type", sortKey: "contentType" },
-    { content: "Created at", sortKey: "createdAt" },
-    { content: "Size", className: "u-align--right" },
     {
-      content: "Used by",
+      content: NAME_COL,
+      sortKey: "name",
+      style: { width: `${COLUMN_WIDTHS[NAME_COL]}px` },
+    },
+    {
+      content: POOL_COL,
+      sortKey: "pool",
+      style: { width: `${COLUMN_WIDTHS[POOL_COL]}px` },
+    },
+    {
+      content: TYPE_COL,
+      sortKey: "type",
+      style: { width: `${COLUMN_WIDTHS[TYPE_COL]}px` },
+    },
+    {
+      content: CONTENT_TYPE_COL,
+      sortKey: "contentType",
+      style: { width: `${COLUMN_WIDTHS[CONTENT_TYPE_COL]}px` },
+    },
+    {
+      content: CREATED_AT_COL,
+      sortKey: "createdAt",
+      style: { width: `${COLUMN_WIDTHS[CREATED_AT_COL]}px` },
+    },
+    {
+      content: SIZE_COL,
+      className: "u-align--right",
+      style: { width: `${COLUMN_WIDTHS[SIZE_COL]}px` },
+    },
+    {
+      content: USED_BY_COL,
       sortKey: "usedBy",
       className: "u-align--right used_by",
+      style: { width: `${COLUMN_WIDTHS[USED_BY_COL]}px` },
     },
     {
       content: "",
       className: "actions",
       "aria-label": "Actions",
+      style: { width: `${COLUMN_WIDTHS[ACTIONS_COL]}px` },
     },
   ];
 
   const filteredVolumes = volumes.filter((item) => {
-    if (!filters.queries.every((q) => item.name.toLowerCase().includes(q))) {
+    // Do not show snapshot volumes at all
+    if (item.name.includes("/")) {
```

Comment:
Probably better to put this in a tiny helper function `isSnapshot(volume: LxdVolume)`, so the code gets speaking and doesn't need a comment here. Also good to reuse in other places.